src/browser-installer/run.ts

```diff
@@ -1,21 +1,27 @@
 import type { ChildProcess } from "child_process";
-import { Driver, type SupportedDriver } from "./utils";
+import { installBrowser } from "./install";
+import { Browser, type SupportedBrowser } from "./utils";
 
 export const runBrowserDriver = async (
-    driverName: SupportedDriver,
+    browserName: SupportedBrowser,
     browserVersion: string,
     { debug = false } = {},
 ): Promise<{ gridUrl: string; process: ChildProcess; port: number }> => {
-    switch (driverName) {
-        case Driver.CHROMEDRIVER:
+    const installBrowserOpts = { shouldInstallWebDriver: true, shouldInstallUbuntuPackages: true };
+
+    await installBrowser(browserName, browserVersion, installBrowserOpts);
```

Comment:
As now we need to install browser before running its driver, we expect "browserName" instead of "driverName" in order to pass "browserName" to "installBrowser"